src/pages/collections/_type/_.vue

```diff
@@ -164,7 +167,9 @@
         entity: state => state.entity.entity,
         page: state => state.entity.page,
         relatedEntities: state => state.entity.relatedEntities,
-        recordsPerPage: state => state.entity.recordsPerPage
+        recordsPerPage: state => state.entity.recordsPerPage,
+        featuredSetId: state => state.entity.featuredSetId,
+        pinned: state => state.entity.pinned
```

Comment:
I don't see `featuredSetId` and `pinned` being used in this component. Are they necessary?

Reply:
You are right, I  removed them